### PR TITLE
VACMS-37630: Add support for action links.

### PR DESCRIPTION
**Pull request by msbtterswrth**
## Description

Relates to #[37630](https://github.com/department-of-veterans-affairs/va.gov-team/issues/37630)
related to https://github.com/department-of-veterans-affairs/va.gov-cms/issues/6604

## Testing done
Edited a page and added the action links class. Confirmed on page save the class remains.

## QA steps
Edit any page with a rich text field, add a link, edit source and add the class for the action links. Save the page and confirm the class appears in the code when inspected. 

As cms user
1. I can edit `/node/33339/edit` 
2. scroll down to a rich text paragraph and click 'source'
3. add the classes `vads-c-action-link--blue` and `vads-c-action-link--green` to two links and 
  - [x] check preview experience for that node
4. trigger a content release and 
  - [x] check the web version of that node.

### Definition of Done

- [ ] Documentation has been updated, if applicable.
- [ ] Tests have been added if necessary.
- [ ] Automated tests have passed.
- [ ] Code Quality Tests have passed.
- [ ] Acceptance Criteria in related issue are met.
- [ ] Manual Code Review Approved.

### Select Team for PR review

- [ ] `Platform CMS Team`
- [ ] `Sitewide CMS Team`
  - [ ] `⭐️ Content ops`
  - [ ] `⭐️ CMS Experience`
  - [ ] `⭐️ Offices`
  - [ ] `⭐️ Product Support`

### Is this PR blocked by another PR?

- [ ] `DO NOT MERGE`

### Does this PR need review from a Product Owner

- [ ] `Needs PO review`

### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
- [ ] Yes, and it's written in issue ____ and queued for publication.
  - [ ] Merge and ping @ rachel-kauff so she's ready to publish after deployment
- [ ] Yes, but it hasn't yet been written
  - [ ] Don't merge yet -- ping @ rachel-kauff to prompt her to write and queue content
- [ ] No announcement is needed for this code change.
  - [ ] Merge & carry on unburdened by announcements

### CODE REVIEW

**Review by ethanteague — APPROVED** [review]
:+1:

**Review by cweagans — DISMISSED** [review]

**Comment by msbtterswrth** [comment]
@kevwalsh good spot, totally missed that duplication! Should be all set now.

**Review by kevwalsh — COMMENTED** [review]
Code looks good. Can we write QA steps to test this end to end?  Eg

1. Edit one of a node with rich text field
2. add the classes to two links 
3. Check preview experience for that node
4. Trigger a content release and check the web version of that node.

i'd rather this not wait until someone on another team tries it to find out if it fails. 

i won't have time to QA today unfortunately.

**Review by ethanteague — APPROVED** [review]
:+1:

File: config/sync/editor.editor.rich_text.yml [hunk 1]
```diff
@@ -45,10 +45,10 @@ settings:
     drupallink:
       linkit_enabled: true
       linkit_profile: default
-    language:
-      language_list: un
     stylescombo:
       styles: "p.va-address-block|Address Block\r\na.usa-button|Default link button\r\na.usa-button-primary.va-button-primary|Primary link button\r\na.usa-button.usa-button-secondary|Secondary link button\r\n"
+    language:
```
Comment (swirtSJW) [inline config/sync/editor.editor.rich_text.yml:50]: Are you running the latest drupal.  Seems odd that this un-alpha sorted.

**Review by swirtSJW — APPROVED** [review]
QA steps check out.
The classes persist beyond save and function on the FE.

This is just a stop-gap measure and should follow up with https://github.com/department-of-veterans-affairs/va.gov-cms/issues/6604 ASAP

**Pull request merged**